src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -616,4 +616,59 @@
     it { expect(assigns(:default_request_type)).to eq('my_type') }
     it { expect(assigns(:default_request_state)).to eq('my_state') }
   end
+
+  describe 'GET #create' do
+    before do
+      login user
+      request.env["HTTP_REFERER"] = root_url # Needed for the redirect_to :back
+    end
+
+    shared_examples "a valid project saved" do
+      it { expect(flash[:notice]).to start_with("Project '") }
```

Comment:
That doesn't seem to be sufficient. If the flash message would be "Project couldn't be saved" or "Project not found" it would still pass